Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/FleetBookingInformation.yaml

```
@@ -41,3 +41,34 @@ FleetBookingInformation:
+  constraints:
+    id: PrimaryKey
+    mainAssignmentId: SecondaryKey
+
+  queries:
+    findAllByMainAssignmentId:
+      kvFunction: findAllWithKV
+      where: mainAssignmentId
+
+  extraOperations:
+    - EXTRA_QUERY_FILE
+    - GENERATE_INDEXES
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)